testfourierSeries.m

```diff
@@ -0,0 +1,53 @@
+ t = 0.1:0.1:3;
+xt = 1+cos(2*pi*t)/4 + cos(2*pi*t*2)/2 + cos(2*pi*t*3)/3;
+plot(t,xt);
+%fft(xt)
+%plot(fft(xt))
+coefficients = fourierSeries(xt);
+oSize = length(xt);
+xtcheck = invFourier(coefficients, t);
+hold on
+plot (t,xtcheck+0.1)
+T1 = false;
+if is_appEq(xt,xtcheck);
+    fprintf("First Test Passed.\n")
+    T1 = true;
+end
+edgecase = [1,3,3,3,3,3,3,2,3,2,1,3,1,3,3,3,3,3,3,2,3,2,1,3];

```

Comment:
you don't need to say edge case. usually when we test we have each test case be its own function and we name the function a test case (for example "happyPathOne" , "repeatingDuplicatesPeriod", "periodOfOne" to denote the case in the function declaration etc)